superset/migrations/versions/c53bae8f08dd_add_saved_query_foreign_key_to_tab_state.py

```diff
@@ -0,0 +1,42 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+"""add_saved_query_foreign_key_to_tab_state
+Revision ID: c53bae8f08dd
+Revises: aea15018d53b
+Create Date: 2021-12-15 15:05:21.845777
+"""
+
+# revision identifiers, used by Alembic.
+revision = "c53bae8f08dd"
+down_revision = "bb38f40aa3ff"
```

Comment:
this revision doesn't match the revision on line 19, fyi

Reply:
Yeah, there have been two migrations since this one, it is updated and correct in the https://github.com/apache/superset/pull/17925
when that gets migrated I am going to rebase that onto this PR